game_start/TODO.txt

```diff
@@ -0,0 +1,6 @@
+- add score
```

Comment:
Tutaj założyłem nowy plik z pomysłami co można jeszcze dorobić, żeby poćwiczyć. Jakbyś miał pytania to śmiało :)